### PR TITLE
Custom Thank You page shortcode

### PR DESCRIPTION
**Pull request by blairlierman**
After a donation is made, the iDonate embed will redirect to a custom Thank You page, passing information as query parameters. 

This change adds a new enclosing shortcode that can be used to read those parameters and add them to a post using template tags (```{{ }}``` ). 
  * Values will be replaced in the template if found in the parameters or removed if not found.
  * A simple ```{{if <value>}}<section>{{end}``` conditional is supported to hide sections if the value is empty or non-existent

### CODE REVIEW

**Comment by blairlierman** [comment]
@jeremyfelt #reviewmerge please

File: includes/class-wsuwp-shortcode-thankyoupage.php [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+				<em> Fax: 509-335-5903</em><br />
+				<em> E-mail: foundation@wsu.edu</em><br />
+			</p>
+			';
+		}
+
+		$content = $this->replace_conditionals( $content, $query_params );
+
+		foreach ( $query_params as $key => $value ) {
+			$content = str_replace( '{{' . $key . '}}', "$value", $content );
```
Comment (jeremyfelt) [inline includes/class-wsuwp-shortcode-thankyoupage.php]: It doesn't look like any of these params are escaped before being added to the content for output. As they all appear to be standard text data and don't require HTML tags, I'd replace with `esc_html( $value )`.

**Comment by blairlierman** [comment]
@jeremyfelt I have added the escaping and pushed the update to this PR.

**Comment by jeremyfelt** [comment]
Looks good!

**Pull request merged**